snap/snapcraft.yaml

```diff
@@ -207,6 +207,8 @@ parts:
     # Last upstream version which doesn't make use of newer cmake features we can't
     # support on 16.04
     source-commit: 1f456938b36f02404830c2e831e328d8ba8d30cd
+    cmake-parameters:
+      - -DSWIFTSHADER_WARNINGS_AS_ERRORS=FALSE
```

Comment:
This option name is used in the latest master branch version of swiftshader. In the stated source-commit version it is called WARNINGS_AS_ERRORS. 

```
+ cmake /home/ubuntu/anbox/parts/swiftshader/src/ -DSWIFTSHADER_WARNINGS_AS_ERRORS=FALSE
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- SPIRV-Tools: nosetests was not found - python support code will not be tested
-- Found PythonInterp: /usr/bin/python3.8 (found version "3.8.2")
-- Did not find googletest, tests will not be built. To enable tests place googletest in '<spirv-dir>/external/googletest'.
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    SWIFTSHADER_WARNINGS_AS_ERRORS
```

So, you need to either change the option name or remove the "source-commit" and use the latest master branch.

Reply:
@salvprest Can you clean this up?